sessions/assemblies.qmd

```diff
@@ -45,62 +47,59 @@ Let’s first move to the root working directory to run all analyses:
 cd /home/training/Assembly/
 ```
 
-You will find all inputs needed for assemblies in the `reads` folder. 
-
-:::{.callout-note}
-If anything goes wrong during the practical, you will find assembly backups 
-for all steps in the respective ".bak" folders. 
-:::
+You will find all inputs needed for assemblies in the "`reads`" folders (both short and long). 
 
 # Short-reads assemblies: metaSPAdes
 For short reads, we will use [SPAdes - St. Petersburg genome Assembler](https://github.com/ablab/spades), a suite of assembling tools containing different assembly pipelines. 
-For metagenomic data, we will the more metagenomi-specific side of SPAdes - metaSPAdes.
-metaSPAdes offers many options that fit your preferences differently, mostly depending on the type of data you are willing to assemble.
+For metagenomic data, we will the more metagenomic-specific side of `SPAdes`: `metaSPAdes`.
+
+metaSPAdes offers many options that fit your preferences differently, mostly depending on the type of data you want to assemble.
 To explore them, type `metaspades.py -h`. 
 Bear in mind that options will differ when selecting different tools (e.g. spades.py) and they should be tuned according to the input dataset and desired outcome.
 
 :::{.callout-tip}
 The default metaSPAdes pipeline executes an error correction step on the input fastqs.
-Since reads have already been polished in the past step, you can run metaspades without the error correction step.
+Since reads have already been polished in the previous step, you could run metaspades without the error correction step.
 
 You can see available metaspades parameters by typing the following: 
 ```bash
 metaspades.py -h 
 ```
 :::
 
-:::{.callout-warning}
-This execution should be able to run on a 4-core, 32 GB ram laptop. However, the command above will NOT work on your VMs to prevent overload on the system if all VM users are running this command at the same time. If you are willing to launch this, the execution will take up to 20 minutes. 
+:::{.callout-important}
+This execution should be able to run on a 4-core, 32 GB ram laptop. However, we kindly ask you to NOT launch the execution on your VMs to prevent overload on the system if all VM users are running this command at the same time. The command will work if assembled correctly, but we kindly ask you to run it at a different moment. The execution would take around 15 minutes. 
```

Comment:
Is this system overload point still correct, when we are running on AWS? 🤔 Or is it just if they have other things open/running/using RAM on their individual VMs it'll cause problems? (Not too important, just wondering)